test/multiple-matrix.json

```diff
@@ -28,6 +28,7 @@
               "_testId": "urn:uuid:7d556c3e-3a4a-48da-bb11-a357c4ddb7f9",
               "_events": {},
               "_eventsCount": 1,
+              "link": "https://example.com/",
```

Comment:
This leave link hanging out with the mocha test properties, I think it's better to move it inside of cell so `cell.link` then the link can be used with the rowId or columnId in fact we should even row something like rowLink columnLink or something like that in the future, but just link for right now seems fine. Has this been run with an actual test suite?

Reply:
It's not per-cell, but per-test... Though, in thinking of this more, there really should be a "metadata" space we populate similar to how https://github.com/mongodb-js/metamocha adds it's data.